docs/tutorial_email_generator.md

```diff
@@ -0,0 +1,225 @@
+# Evaluating an Email Generator
```

Comment:
We can do this separately, but it would be pretty cool to have a UI like Koki's Q&A app that shows a bunch of the langcheck metrics next to the generated email

Reply:
Could also add a link to a Colab notebook so people can try the tutorial easily, but I think having a UI would be even better

Reply:
I think a colab would be nice so that users can change the prompts and reference outputs easily. Having a UI in addition to that would be cool too, but I'll punt on that for now!

Reply:
added a colab and linked it at the top of the tutorial!

Reply:
Nice!!